components/homme/src/share/gllfvremap_mod.F90

```diff
@@ -2175,6 +2208,29 @@ subroutine limiter_clip_and_sum(spheremp, qmin, qmax, dp, q)
     call limiter1_clip_and_sum(np*np, spheremp, qmin, qmax, dp, q)
   end subroutine limiter_clip_and_sum
 
+  subroutine ref2plane(corners, a, b, plane)
```

Comment:
i assume this is needed for SL?

Reply:
This is for the metric terms in dynamics-physics grid remap. This PR is to support pg2 in planar mode.